frontend/package.json

```diff
@@ -32,104 +32,106 @@
+    "url-loader": "1.0.1",
+    "webpack": "4.16.1",
+    "webpack-bundle-analyzer": "2.13.1",
+    "webpack-cli": "3.0.8",
+    "webpack-dev-server": "3.1.4",
+    "webpack-merge": "4.1.3",
+    "why-did-you-update": "0.1.1"
   },
   "dependencies": {
-    "@turf/bbox": "^5.1.5",
-    "babel-polyfill": "^6.26.0",
-    "body-parser": "^1.18.2",
-    "chroma-js": "^1.3.6",
-    "classnames": "^2.2.5",
-    "d3": "^4.13.0",
-    "d3-geo": "^1.10.0",
-    "d3-geo-projection": "^2.3.2",
-    "d3-plugins-sankey": "^1.2.1",
-    "dotenv": "^4.0.0",
-    "downshift": "^1.31.12",
-    "find-parent": "^2.0.2",
-    "highlight-words-core": "^1.2.0",
-    "leaflet": "^1.3.1",
+    "@turf/bbox": "6.0.1",
+    "babel-polyfill": "6.26.0",
+    "body-parser": "1.18.3",
+    "chroma-js": "1.3.7",
+    "classnames": "2.2.6",
+    "d3": "5.5.0",
+    "d3-request": "1.0.6",
+    "d3-selection": "1.3.0",
+    "d3-geo": "1.10.0",
+    "d3-geo-projection": "2.4.0",
+    "d3-plugins-sankey": "1.2.1",
+    "dotenv": "6.0.0",
+    "downshift": "2.0.16",
+    "find-parent": "2.0.2",
+    "highlight-words-core": "1.2.0",
+    "leaflet": "1.3.3",
     "leaflet.utfgrid": "git+https://github.com/danzel/Leaflet.utfgrid.git",
-    "lodash": "^4.17.5",
-    "plyr": "^3.0.3",
-    "prop-types": "^15.6.1",
-    "qs": "^6.5.1",
-    "rangetouch": "^1.0.5",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0",
-    "react-input-autosize": "^2.2.1",
-    "react-placehodl": "^1.0.3",
-    "react-redux": "^5.0.7",
+    "lodash": "4.17.10",
+    "plyr": "3.3.22",
+    "prop-types": "15.6.2",
+    "qs": "6.5.2",
+    "rangetouch": "1.0.5",
+    "react": "16.4.1",
+    "react-dom": "16.4.1",
+    "react-input-autosize": "2.2.1",
+    "react-placehodl": "1.0.3",
+    "react-redux": "5.0.7",
     "react-siema": "git+https://github.com/sorodrigo/react-siema.git",
-    "react-simple-maps": "^0.12.0",
-    "react-tether": "^0.6.1",
-    "react-transition-group": "^2.2.1",
-    "redux": "^3.7.2",
-    "redux-first-router": "^0.0.4-rudy",
-    "redux-first-router-link": "^1.4.2",
-    "redux-first-router-restore-scroll": "^1.2.1",
-    "redux-logger": "^3.0.6",
-    "redux-thunk": "^2.2.0",
-    "remark": "^9.0.0",
-    "remark-react": "^4.0.1",
-    "reselect": "^3.0.1",
-    "tooltip.js": "^1.2.0",
-    "topojson": "^3.0.2",
-    "turf-linestring": "^1.0.2",
-    "whatwg-fetch": "^2.0.4"
+    "react-simple-maps": "0.12.1",
+    "react-tether": "1.0.1",
+    "react-transition-group": "2.4.0",
+    "redux": "4.0.0",
+    "redux-first-router": "0.0.9-rudy",
+    "redux-first-router-link": "1.4.2",
+    "redux-first-router-restore-scroll": "1.2.1",
+    "redux-logger": "3.0.6",
+    "redux-thunk": "2.3.0",
+    "remark": "9.0.0",
+    "remark-react": "4.0.3",
+    "reselect": "3.0.1",
+    "tooltip.js": "1.2.0",
+    "topojson": "3.0.2",
+    "turf-linestring": "1.0.2",
+    "whatwg-fetch": "2.0.4"
   },
   "engines": {
-    "node": "8.2.1"
+    "node": "10.6.0"
```

Comment:
This should always be `>= x.x.x` to a reasonable version. And by reasonable I mean lowest working version.

Reply:
:+1: